test/Microsoft.Net.Http.Headers.Tests/ContentDispositionHeaderValueTest.cs

```diff
@@ -560,6 +560,34 @@ public void ContentDispositionHeaderValue_ParseInvalid_Throws(string input)
             Assert.Throws<FormatException>(() => ContentDispositionHeaderValue.Parse(input));
         }
 
+        [Fact]
+        public void HeaderNamesWithQuotes_ExpectNamesToNotHaveQuotes()
+        {
+            var contentDispositionLine = "form-data; name =\"dotnet\"; filename=\"example.png\"";
+            var expectedName = "dotnet";
+            var expectedFileName = "example.png";
+
+            var result = ContentDispositionHeaderValue.Parse(contentDispositionLine);
+
+            Assert.Equal(expectedName, result.Name);
+            Assert.Equal(expectedFileName, result.FileName);
+        }
+
+        [Fact]
+        public void HeaderNamesWithQuotes_CheckSettersToNotHaveQuotes()
```

Comment:
I found that the test I wanted already exists:
https://github.com/aspnet/HttpAbstractions/blob/98da5378bd3383d8590b71e36e4f7a8f76d0e41f/test/Microsoft.Net.Http.Headers.Tests/ContentDispositionHeaderValueTest.cs#L130